src/plugins/recordTypes/media/fields.js

```diff
@@ -0,0 +1,91 @@
+import { defineMessages } from 'react-intl';
+
+export default (configContext) => {
+  const {
+    AutocompleteInput,
+    TextInput,
+    OptionPickerInput,
+  } = configContext.inputComponents;
+
+  const {
+    configKey: config,
+  } = configContext.configHelpers;
+
+  const {
+    DATA_TYPE_INT,
+  } = configContext.dataTypes;
+
+  return {
+    document: {
+      'ns2:media_botgarden': {
+        [config]: {
+          service: {
+            ns: 'http://collectionspace.org/services/media/local/botgarden',
+          },
+        },
+        handwritten: {
+          [config]: {
+            messages: defineMessages({
+              name: {
+                id: 'field.media_botgarden.handwritten.name',
+                defaultMessage: 'Handwritten label',
+              },
+            }),
+            view: {
+              type: OptionPickerInput,
+              props: {
+                source: 'booleans',
```

Comment:
This can't use the `booleans` list, because the values in that list are `true` and `false`, not `Unknown`, `yes`, and `no`. Remember that the values in 5.0 must match the ids in 4.1 config, or else we'll create really bad data inconsistency problems. Create a new option list for this.